mirrord/agent/src/steal/ip_tables.rs

```diff
@@ -117,6 +117,17 @@ pub struct IPTablesWrapper {
     tables: Arc<iptables::IPTables>,
 }
 
+/// wrapper around iptables::new that uses nft or legacy based on env
+pub fn new_iptables() -> Result<iptables::IPTables, Box<dyn std::error::Error>> {
+    if let Some(val) = std::env::var("MIRRORD_AGENT_NFTABLES")
+        && val.lowercase() == "true"
+    {
+        iptables::new_with_cmd("/usr/sbin/iptables-nft")
+    } else {
+        iptables::new_with_cmd("/usr/sbin/iptables-legacy")
+    }
+}
```

Comment:
```suggestion
pub fn new_iptables() -> iptables::IPTables {
    if let Some(val) = std::env::var("MIRRORD_AGENT_NFTABLES")
        && val.lowercase() == "true"
    {
        iptables::new_with_cmd("/usr/sbin/iptables-nft")
    } else {
        iptables::new_with_cmd("/usr/sbin/iptables-legacy")
    }
    .expect("IPTables initialization may not fail!")
}
```

And remove the `unwrap`s from the other pieces of code. We're just `unwrap`ping this anyway right?